actors/verifreg/tests/verifreg_actor_test.rs

```diff
@@ -488,6 +488,7 @@ mod claims {
                     make_claim(2, alloc2, sector_id(provider, 1000), 1500),
                     make_claim(3, alloc3, sector_id(provider, 1000), 1500),
                 ],
+                128 * 3,
```

Comment:
nit: more my bug than yours but we should demagicify 128 as the piece size with a constant of something

Reply:
Agreed, I'm planning to come back for this after moving more helper methods to the harness module, and making size a parameter.